### PR TITLE
support for all .eth username resolution

### PR DESCRIPTION
**Pull request by yenda**
ACTION BEFORE MERGE: replace "thisisme" by "stateofus" for name resolution without domain name

- resolve all eth usernames
- resolve stateofus.eth without domain

### Steps to test:
- barry should resolve as a username and open a chat with barry, the Wellit Impossible Crocodile

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/add_new/new_chat/events.cljs [hunk 1]
```diff
@@ -3,26 +3,29 @@
...
-                                                    chain)
-                                     :ens-name new-identity
-                                     :cb #(re-frame/dispatch [:new-chat/set-new-identity %])}})
-       {:db (assoc db
-                   :contacts/new-identity       new-identity
-                   :contacts/new-identity-error new-identity-error)}))))
+                                     :registry (get ens/ens-registries chain)
+                                     :ens-name (if (ens/is-valid-eth-name? new-identity)
+                                                 new-identity
+                                                 (str new-identity ".thisisme.eth"))
```
Comment (jeluard) [inline src/status_im/ui/screens/add_new/new_chat/events.cljs]: Should be stateofus?

**Review by jeluard — CHANGES_REQUESTED** [review]

**Review by jeluard — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/add_new/new_chat/events.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/ui/screens/add_new/new_chat/events.cljs]: Will be changed after test

**Comment by asemiankevich** [comment]
@yenda so basically what should be expected in this PR? i dont get how `barry` could be resolved to some username.
So to do this i shoud register barry.stateofus.eth and then when create a private chat i can type only `barry` into this username field right? What about wallet send?

**Comment by yenda** [comment]
@asemiankevich barry is already on mainnet. It's `barry.thisisme.eth` but you can just type `barry` and it will be resolved. Before merging I will switch `thisisme.eth` by `stateofus.eth` to resolve these names instead. It can also resolve any .eth name that has a pubkey registered if you type the full domain.

What should be tested is that you can resolve a `.thisisme.eth` name without the domain (which means it will work for `stateofus.eth` once it is deployed) and any name with the domain

**Comment by asemiankevich** [comment]
can resolve `barry`, `Barry`, `barry.thisisme.eth` on mainnet and open chat with Wellit Impossible Crocodile

**Comment by jeluard** [comment]
@yenda Can you replace `thisisme` ?

**Pull request merged**